Libraries/LibJS/Runtime/Error.h

```diff
@@ -39,6 +39,8 @@ class Error : public Object {
     const FlyString& name() const { return m_name; }
     const String& message() const { return m_message; }
 
+    void set_name(FlyString& name) { m_name = name; }
```

Comment:
Since we don't intend to mutate the `FlyString` itself here:

```suggestion
    void set_name(const FlyString& name) { m_name = name; }
```